### PR TITLE
Build SEM-specific default download experience (Fixes #6829)

### LINKED ISSUE #6829: Build SEM-specific default download experience
## Description

Create a separate version of /new for paid funnel use

[Brief](https://docs.google.com/document/d/1Pm2xZTb337ZC-FbB89PoPL_8YHNfBq-0qhFiRg16YRo/edit#)
[Airtable listing](https://airtable.com/tbl1JN1aCB5UXpjvE/viwSPe2isp9ualv6O/recjJ3Bx5yWLV7U77)
[Context Doc](https://docs.google.com/document/d/1PucezGPizQFe2bZemAaISiig1bueFPvkEPhB8biLCEM/edit?usp=sharing)

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | 1 day |
| PM | 1/2 day |
| ENG | 2 & 1/2 days |
| SEO | 3 days |
---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] Page is build wholly using Protocol components
- [x] Page comes as close to replicating existing /new as possible, within Protocol constraints (e.g. same images, background, and appx. same layout if possible)
- [x] Page lives at /firefox/campaign
- [x] Page uses a copy of existing /new .lang file which (requires conversation with l10n team)
- [x] Canonicalize the page to /new so this page is not indexed
- [x] Build for DE and en-GB locales

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] 

#### Tasks
- [x] Understand the l10n approach
- [x] Build the page(s)
- [x] Build the lang file(s)
- [x] Launch on demo
- [x] Ensure new page(s) respond to DE and en-GB requests

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a Protocol-based, simplified version of the `/firefox/new/` page at `/firefox/campaign/`.
- This new page looks as close as possible to the existing /new/, but without the complex features aimed only at Firefox users.
- The idea is that this page will be completely independent of _any_ changes that happen to /new.

Note: this blocks #6831, so marking this as a P1.

## Issue / Bugzilla link
#6829

## Testing
Demo: https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/campaign/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/campaign.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% add_lang_files "firefox/campaign" %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/campaign.html]: @peiying2 hi, I have a slightly unusual request here.

This PR is adding a new `/firefox/campaign/` page, but the strings are all from the current `/firefox/new/` page. The idea is that this new page will be completely independent to `/new`, so we would like to create a duplicate l10n file for it. There is no requirement to translate any new strings, but we would like to copy all the strings from `firefox/new/quantum` to this new lang file.

Is this something you might be able to help us with? I could do it manually and submit a PR to the l10n repo, but there might be a quicker way I don't know about?

**Comment by peiying2** [comment]
Apart from making a copy, we need to add the new file name to a langchecker file that tracks all the active files, the locales it supports, deadlines, priority, and activation when a file is fully translated, so it can go the production. So it’s not just the string extraction or populating the files. Without it being registered in langcheck file system, it won’t show up in Pontoon or other project dashboard

**Comment by alexgibson** [comment]
> Apart from making a copy, we need to add the new file name to a langchecker file that tracks all the active files, the locales it supports, deadlines, priority, and activation when a file is fully translated, so it can go the production. So it’s not just the string extraction or populating the files. Without it being registered in langcheck file system, it won’t show up in Pontoon or other project dashboard

Ok how can I help to do that?

File: bedrock/firefox/templates/firefox/campaign.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/campaign.html]: I opened a pull request for the langfile here https://github.com/mozilla-l10n/www.mozilla.org/pull/277

**Comment by alexgibson** [comment]
Ok, the lang file is in place so this is now ready for code review.

File: bedrock/firefox/templates/firefox/campaign.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      class='mzp-has-image mzp-u-no-shape mzp-t-firefox mzp-t-dark',
+      include_cta=True,
+      image_url='firefox/new/browser.png',
+      include_highres_image=True,
+      heading_level=1
+    ) %}
+      {{ download_firefox(alt_copy=_('Download Now'), download_location='primary cta') }}
+    {% endcall %}
+  </div>
+  <div class="features">
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/campaign.html]: Semantic nit: would this be better as a section tag?

File: bedrock/firefox/templates/firefox/campaign.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="main-content">
+    <div class="mzp-l-content">
+      <div class="header-logos">
+        <h2 class="logo-firefox"><a href="{{ url('firefox') }}" title="{{ firefox_logo_title }}">{{ high_res_img('logos/firefox/logo-quantum-wordmark-white.png', {'alt': firefox_logo_title, 'width': '133', 'height': '50'}) }}</a></h2>
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/campaign.html]: Semantic nit: what do we think about `<header class="header-logos">`?

File: bedrock/firefox/templates/firefox/campaign.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+{% block canonical_urls %}
+  {# the SEM campaign page should set canonical to the /firefox/new/ page. #}
+  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/">
+{% endblock %}
+
+{% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
+
+{%- block page_title -%}{{_('Free Web Browser')}}{%- endblock -%}
+
+{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!')}}{% endblock %}
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/campaign.html]: Nit: web is capitalized in "a free Web browser"

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/campaign.html]: These are all existing strings and heavily translated, so I'm not gonna change this one at this time. Good call though 👍

**Comment by alexgibson** [comment]
Thanks @amychurchwell, updated

**Pull request merged**